config/routes.rb

```diff
@@ -26,13 +26,19 @@
   get "/cart/delete/:id", to: "carts#delete_item", as: :delete_order_item
   post "/cart/update/:id", to: "carts#update_item", as: :update_order_item
   get "/contact", to: "landing#contact"
+  get "/about", to: "landing#about"
+  get "/blog", to: "landing#blog"
+  get "/blog/:id", to: "landing#single_post"
+  get "/faq", to: "landing#frequently_asked_questions"
```

Comment:
The routes are getting larger and a lot less organized. You could make use of scopes to make things a bit cleaner.

Make adjustments to the routes you created and if you can other routes in the file.
@andela-tpeters 

Check out this resource on routing 
http://edgeguides.rubyonrails.org/routing.html

Additional resource on scoping http://api.rubyonrails.org/classes/ActionDispatch/Routing/Mapper/Scoping.html